elastic/kibana
# [Ingest Manager] Ensure we trigger agent policy updated event when we bump revision.

Resolve https://github.com/elastic/kibana/issues/78804 #78013

When a policy is updated we should create a new agent action, so agent can get the new policy.
That policy is generated when in the agent policy service we trigger an `updated` event, that event trigger was missing.


## How to test this

1. Enroll an agent in the default config => the config revision should be `n`
1. if you go on the listing page you should see the agent as online
1.  Update kibana url | or es url in the settings pannel (just click on save do not update values)
1. If you refresh the agent listing your agent should be marked as out of date, as the agent action is not generated, after the fix it should online and up to date.
